yt_dlp/extractor/soundcloud.py

```diff
@@ -352,12 +358,14 @@ def genNumBlock():
 
         query = self._API_AUTH_QUERY_TEMPLATE % self._CLIENT_ID
         login = sanitized_Request(self._API_AUTH_URL_PW % query, json.dumps(payload).encode('utf-8'))
-        response = self._download_json(login, None)
-        self._access_token = response.get('session').get('access_token')
+        response = self._download_json(login, None, fatal=False)
+        if response is not False:
+            self._access_token = response.get('session').get('access_token')
         if not self._access_token:
-            self.report_warning('Unable to get access token, login may has failed')
+            self.report_warning('Unable to get access token, login may has failed. Try cookies/authorization header instead')
```

Comment:
why? This code is being commented out, no?

Reply:
I actually added that as one of the earliest changes and forgot about it. The whole block isn't removed since its logic is still applicable in the future when the missing parameters can be generated :)

Reply:
> The whole block isn't removed since its logic is still applicable in the future when the missing parameters can be generated

Yeah, I got that :D